pages/common/ncu.md

```diff
@@ -0,0 +1,28 @@
+# ncu
+
+> Find newer versions of package dependencies and check outdated npm packages locally or globally.
+> More information: <https://github.com/raineorshine/npm-check-updates>.
+
+- Install npm package `node-check-updates` globally:
+
+`npm install -g node-check-updates`
```

Comment:
Is there any other page with an example of how to install the program?
```suggestion
```

Reply:
Yep, We don't generally include installation commands to a page.